lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -82,6 +83,24 @@ def main():
     else:
         config = std_config
 
+    increase_nsb = False
+    increase_psf = False
+    if "image_modifier" in config:
+        imconfig = config["image_modifier"]
```

Comment:
Why make all of these config objects local variables? Why not just use the config where the function is called?

Reply:
Just tried, but I don't see the advantage, the code where the function is called becomes much less readable.
Besides, I tested this as it is - don't underestimate my capacity to introduce a bug while doing an apparently harmless change.